app/helpers/settings_schedule_helper.rb

```diff
@@ -0,0 +1,116 @@
+module SettingsScheduleHelper
+  private
+
+  def settings_schedule_summary(schedule, exp_table, object_class, object_name)
+    automation_request_check = schedule.sched_action[:method] == 'automation_request'
+    summary = [
+      settings_schedule_basic_info(schedule),
+      settings_schedule_filter_details(schedule, exp_table),
+    ]
+    if automation_request_check
+      summary.push(settings_schedule_object_details(schedule))
+      summary.push(settings_schedule_object_attribute(object_class, object_name))
+      summary.push(settings_schedule_attribute_value_pairs(schedule)) if schedule.filter[:ui][:ui_attrs].present?
+    end
+    safe_join(summary)
+  end
+
+  def settings_schedule_basic_info(schedule)
+    data = {
+      :title => _('Basic Information'),
+      :mode  => "settings_schedule_basic_info",
+    }
+
+    rows = [
+      row_data(_('Description'), schedule.description),
+      row_data(_('Active'), schedule.enabled),
+    ]
+
+    if schedule.sched_action[:method] && schedule.sched_action[:method] == "check_compliance"
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Compliance Check')}"))
+    elsif schedule.sched_action[:method] && schedule.sched_action[:method] == "automation_request"
+      rows.push(row_data(_('Action'), _('Automate Task')))
+    else
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Analysis')}"))
+    end
+
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def settings_schedule_filter_details(schedule, exp_table)
+    data = {
+      :title => _('Filter'),
+      :mode  => "settings_schedule_filter_details",
+    }
+
+    rows = []
+    unless schedule.sched_action[:method] == "automation_request"
+      if schedule.miq_search
+        search = schedule.miq_search
+        filter_data = search.search_type == "user" ? `_("My Filter: ") #{search.description}` : `_("Global Filter: ") #{search.description}`
```

Comment:
@mahantidinesh7 This is a security issue that is breaking master - backticks in Ruby execute code (unlike in Javascript where they are Strings)

cc @jrafanie

Reply:
From master for this repo:

```
% bundle rake app:test:security

```
== Warning Types ==

Command Injection: 1

== Warnings ==

Confidence: Medium
Category: Command Injection
Check: Execute
Message: Possible command injection
Code: `_("My Filter: ") #{schedule.miq_search.description}`
File: ../manageiq-ui-classic/app/helpers/settings_schedule_helper.rb
Line: 51
```

I think you can just change the backticks to single quotes.

Reply:
cc @jeffibm @MelsHyrule

Reply:
@jrafanie As discussed, let's introduce core's security suite to this repo (and API also), then get that into master to show it's red.  Then we can do a followup PR to show it go green.

Reply:
Ok.let me raise a new PR changing the  backticks to single quotes

Reply:
@Fryguy please check this PR which has the fix https://github.com/ManageIQ/manageiq-ui-classic/pull/8705 and thanks for pointing it out. I am not aware of this security issue in ruby

Reply:
No worries.  Backticks exist in ruby but they are completely different.  They basically shell out to run a command.  This is why it was complaining about a possible command injection. 

```
irb(main):001:0> `date`
=> "Tue Mar 14 10:22:11 EDT 2023\n"
```  

https://ruby-doc.org/core-3.0.2/Kernel.html#method-i-60